src/textual/widgets/_tabbed_content.py

```diff
@@ -26,15 +27,46 @@
 class ContentTab(Tab):
     """A Tab with an associated content id."""
 
-    def __init__(self, label: Text, content_id: str, disabled: bool = False):
+    _PREFIX: Final[str] = "--content-tab-"
+    """The prefix given to the tab IDs."""
+
+    @classmethod
+    def add_prefix(cls, content_id: str) -> str:
+        """Add the prefix to the given ID.
+
+        Args:
+            content_id: The ID to add the prefix to.
+
+        Returns:
+            The ID with the prefix added.
+        """
+        return f"{cls._PREFIX}{content_id}" if content_id else content_id
+
+    @classmethod
+    def sans_prefix(cls, content_id: str) -> str:
+        """Remove the prefix from the given ID.
+
+        Args:
+            content_id: The ID to remove the prefix from.
+
+        Returns:
+            The ID with the prefix removed.
+        """
+        return (
+            content_id[len(cls._PREFIX) :]
+            if content_id.startswith(cls._PREFIX)
+            else content_id
+        )
```

Comment:
Let's drop support for 3.8 so this can be `return content_id.removeprefix(cls._PREFIX)` 😆

Reply:
If only...